### PR TITLE
Update install_prereqs.sh for individual use

### LINKED ISSUE #1191: install prereq.sh script fails to due to locale problem, are you testing it?
rocm5.1.0
It is such a fundamental build error. build instruction clearly tells to use non-working script install_prereqs.sh which fails. Applying recommended export does not work either.

Traceback (most recent call last):
  File "/usr/local/bin/rbuild", line 11, in <module>
    load_entry_point('rbuild==0.0.1', 'console_scripts', 'rbuild')()
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 760, in main
    _verify_python3_env()
  File "/usr/local/lib/python3.6/site-packages/click/_unicodefun.py", line 130, in _verify_python3_env
    " mitigation steps.{}".format(extra)
RuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment. Consult https://click.palletsprojects.com/python3/ for mitigation steps.

This system supports the C.UTF-8 locale which is recommended. You might be able to resolve your issue by exporting the following environment variables:

    export LC_ALL=C.UTF-8
    export LANG=C.UTF-8

### PR DESCRIPTION
**Pull request by causten**
Our [documentation](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX#installing-the-prerequisites) indicates a user with sudo can run the install_prereqs.sh file.  Turns out that the file is not complete enough to run on Ubuntu 18.04/20.04 independently.  I updated the file to resolve the failures.

resolves #1191 

I tested this script independently (as root and as a non root with sudo), Dockerfile, and hip-clang.dockerfile.  All build the source.

### CODE REVIEW

**Comment by pfultz2** [comment]
> I did remove the -s develop in the rebuild statement.

If you remove the develop dependencies it wont work correctly.

File: tools/install_prereqs.sh [hunk 1]
```diff
@@ -4,12 +4,24 @@
...
+
+# Need pip3 and Python headers to build dependencies
+apt update && apt install -y python3-pip python3-dev cmake
+#apt update && apt install -y python3-pip python3-dev cmake
+
+# Needed for cmake to build various pip packages
+pip3 install setuptools wheel
+
+#install rocm-cmake, rocblas and miopen
+apt install -y rocm-cmake rocblas miopen-hip openmp-extras
```
Comment (pfultz2) [inline tools/install_prereqs.sh:19]: This should be combined with the install on line 12.

**Comment by causten** [comment]
> 

Very true.  I have reverted that change

**Review by CharlieL7 — APPROVED** [review]
Works with rocm/dev-ubuntu-20.04:5.0 docker image. Must run from base project folder.

**Review by pfultz2 — APPROVED** [review]

**Pull request merged**